locales/en-US/app.ftl

```diff
@@ -635,7 +635,7 @@ ProfileRootMessage--additional = Back to home
 ## This is the component responsible for handling the service worker installation
 ## and update. It appears at the top of the UI.
 
-ServiceWorkerManager--installing-button = Installing…
+ServiceWorkerManager--applying-button = Applying…
```

Comment:
Out of curiosity, why the change? Is it applying an update, or something else?

Reply:
Yeah, everything has been downloaded already in background. This button will only apply the changes.

Previously I believe I used "installing" because this was easier to understand, however the action button reads `Apply and reload` already so I thought this is also more consistent.